packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx

```diff
@@ -28,98 +27,54 @@ type TrackMetadataFieldsProps = {
   type: 'track'
 }
 
-const TrackMetadataFields = (props: TrackMetadataFieldsProps) => {
-  const [imageProcessingError, setImageProcessingError] = useState(false)
-  const [artworkField, , artworkHelpers] = useField('artwork')
-  const [titleField, titleMeta, titleHelpers] = useField('title')
-  const [genreField, genreMeta, genreHelpers] = useField('genre')
-  const [, moodMeta, moodHelpers] = useField('mood')
-  const [, tagsMeta, tagsHelpers] = useField('tags')
-  const [, descriptionMeta, descriptionHelpers] = useField('description')
-
-  const onDropArtwork = async (selectedFiles: File[], source: string) => {
-    try {
-      let file = selectedFiles[0]
-      file = await resizeImage(file)
-      // @ts-ignore writing to read-only property. Maybe bugged?
-      file.name = selectedFiles[0].name
-      const url = URL.createObjectURL(file)
-      artworkHelpers.setValue({ url, file, source })
-      setImageProcessingError(false)
-    } catch (err) {
-      console.error(getErrorMessage(err))
-      setImageProcessingError(true)
-    }
-  }
+export const TrackMetadataFields = (props: TrackMetadataFieldsProps) => {
+  const { type } = props
 
   return (
     <div className={styles.basic}>
       <div className={styles.artwork}>
-        <UploadArtwork
-          artworkUrl={artworkField.value?.url}
-          onDropArtwork={onDropArtwork}
-          imageProcessingError={imageProcessingError}
-        />
+        <ArtworkField name='artwork' />
       </div>
       <div className={styles.fields}>
         <div className={styles.trackName}>
-          <Input
-            name='name'
-            id='track-name-input'
-            placeholder={`${
-              props.type.charAt(0).toUpperCase() + props.type.slice(1)
-            } Name`}
-            defaultValue={titleMeta.initialValue}
-            characterLimit={64}
-            error={!!titleMeta.error}
-            variant={'elevatedPlaceholder'}
-            onChange={titleHelpers.setValue}
-            onBlur={titleField.onBlur}
+          <TextField
+            name='title'
+            variant={InputV2Variant.ELEVATED_PLACEHOLDER}
+            label={`${formatCapitalizeString(type)} Name`}
```

Comment:
we can probably remove type and just call this 'Track Name' since this is TrackMetadataFields now

Reply:
oh yeah totally, will update